CMakeLists.txt

```diff
@@ -58,6 +58,7 @@ ENDIF ()
 # ======== Code Time ========
 option(PLATFORM "The platform to build for, either \"firmware\" or \"dimos\"" "firmware")
 option(TARGET "The target to build for. Choose between \"binary\" and \"test\"." "binary")
+option(FIRMWARE_TARGET "The firmware target to build for. Choose between \"dev\" and \"chimera\"." "dev")
```

Comment:
can we make it so that firmware builds have chimera and non-chimera targets? achieved by making seperate chimera targets